recipes/gnupg/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "gnupg" %}
+{% set version = "2.2.4" %}
+{% set sha256 = "401a3e64780fdfa6d7670de0880aa5c9d589b3db7a7098979d7606cec546f2ec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CPPFLAGS="-DIN_EXCL_UNLINK=0x04000000 $CPPFLAGS"  # [linux]
+    - export LDFLAGS="-lrt $LDFLAGS"                           # [linux]
+    - ./configure --prefix=$PREFIX
+    - make
+    - make install
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - bzip2
+    - sqlite
+    - libgcrypt
+    - libgpg-error
+    - libksba
+    - libassuan
+    - ntbtls
+    - npth
+  run:
+    - libcxx
```

Comment:
Can you remove `libcxx` from all the recipes?

Reply:
sure, guess its not needed?

Reply:
If you are not using clang from the clangdev conda package, it's not needed.

Reply:
basically added this to have the system bit of `check linkages`

Reply:
libc++ being available is a guarantee on OSX >=10.9 and conda-forge packages only support OSX >=10.9